master/internal/sproto/task.go

```diff
@@ -272,8 +273,12 @@ func (ev *Event) ToTaskLog() model.TaskLog {
 		message = ""
 	}
 
+	level := ptrs.Ptr(model.LogLevelInfo)
+	if ev.Level != nil {
```

Comment:
What is the meaning of "ev.Level" and why do we overwrite "level" with "ev.Level"?

Reply:
Event message severity level. All event messages were logged at LogLevelInfo, including the failure message this PR is fixing.

Reply:
What if "ev.Level" is TRACE or DEBUG, assuming those are valid values?  Are we still OK?

Reply:
let's see what @stoksc thinks.

Reply:
if they are trace or debug, it seems the sender could send them at that level now? and info is a fine default. i think this is ok.